### ISSUE TITLE
cluster_generic.py from f5-openstack-onboard should be ported to this repo and refactored

### ISSUE THREAD

**Issue by pjbreaux** [F5Networks/f5-common-python#209]
The cluster_generic module utilizes functionality from the cluster.py module in this repo to cluster BigIPs. The cluster_generic.py module currently lives https://github.com/F5Networks/f5-openstack-onboard/tree/develop/f5_onboarding

It should be brought into this repo where it makes sense and refactored to use the refactored cluster.py code. So this is dependent upon Issue #208 being completed.

**Comment by zancas** [comment]
I think @pjbreaux 's current work addresses this.

**Comment by wojtek0806** [comment]
Issue Review: Yes this is also a duplicate of #400

**Comment by pjbreaux** [comment]
Yep, duplicate of #400.

**Comment by zancas** [comment]
Agreed.  Duplicate.  Closing.
